toolkit/tools/internal/file/file.go

```diff
@@ -45,13 +45,12 @@ func Move(src, dst string) (err error) {
 
 	src, err = filepath.Abs(src)
 	if err != nil {
-		logger.Log.Errorf("Failed to get absolute path for move source (%s).", src)
-		return
+		return fmt.Errorf("failed to get absolute path for move source (%s)\n%w", src, err)
```

Comment:
nit: Missing `:`.

`"failed to get absolute path for move source (%s):\n%w"`

Reply:
I removed the `:` since `err` will be printed on a newline. Planning to follow this format throughout the toolkit now. If it reads better with `:`, I can add it

Reply:
I think it is better with the `:`